bugbot/rules/crash_small_volume.py

```diff
@@ -181,6 +184,14 @@ def set_autofix(self, bugs):
                 )
                 autofix["keywords"] = {"remove": list(bug["keywords_to_remove"])}
 
```

Comment:
I do not think the whitespace is needed here:
```suggestion
```

Reply:
Removed in 1ae99e364122b2c0b327c4d537cc95f8b7c706e4!